.github/workflows/publishdocker.yml

```
@@ -0,0 +1,25 @@
+    branches: [ "master" ]
+jobs:
+  build:
+    if: github.repository_owner == 'ytti'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: 0
+    - name: Get Release Version
+      id: get_version
+      run: echo "release-version=$(git describe --tags)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)